src/components/EventsList/Actions/Sort.js

```diff
@@ -1,27 +1,32 @@
 import { FormControl, InputLabel, MenuItem, Select } from "@mui/material";
 
-import "../styles.scss";
+import styles from "../EventsList.module.scss";
 
-import { capitalizeWordFirstLetter } from "../../../utils/utils";
+import {
+  capitalizeWordFirstLetter,
+  SELECT_OPTIONS,
+} from "../../../utils/utils";
 
 const Sort = ({ value, defaultVal, handleChange }) => {
-  const options = ["none", "title", "date", "description"];
+  const options = SELECT_OPTIONS;
```

Comment:
you don't need to reassign this. Just Use SELECT_OPTIONS in the code. Also, I would suggest a more suggestive name since is in the utils file. Maybe SORT_FIELDS or SORT_FIELD_OPTIONS so that is clear that those are options for selecting a sorting field value